### PR TITLE
[system] unassign group from user

### LINKED ISSUE #5: [system] remove group from user
We should be able to remove a group from a user.

### PR DESCRIPTION
**Pull request by axelfahy**
Add method to unassign a group from a user. Closes #5

### CODE REVIEW

File: system/edit_queries/user_delete_group.graphql
Comment (TheWanderingKobold) [inline system/edit_queries/user_delete_group.graphql:1]: Could be renamed `user_unassign_group.graphql` or `user_remove_group.graphql`

File: system/utils.go [hunk 1]
```diff
@@ -55,6 +55,43 @@ func (u User) AssignGroup(
 	return finalMap, nil
 }
 
+//go:embed edit_queries/user_delete_group.graphql
+var userDeleteGroupQueryString string
```
Comment (TheWanderingKobold) [inline system/utils.go:59]: Could also be renamed more appropriately

Comment (axelfahy) [reply system/utils.go]: 👍🏻 I have standardized all the names (assign / unassign) in utils.

File: system/edit_queries/user_delete_group.graphql
Comment (axelfahy) [reply system/edit_queries/user_delete_group.graphql:1]: 👍🏻  I have standardized all the queries.

**Review by TheWanderingKobold — APPROVED** [review]
LGTM

**Review by yanover — APPROVED** [review]

**Pull request merged**